VCToken/VCToken/JwsToken.swift

```diff
@@ -65,10 +65,15 @@ public struct JwsToken<T: Claims> {
         self.signature = try signer.sign(token: self, withSecret: secret)
     }
     
-    public func verify(using verifier: TokenVerifying, withPublicKey key: ECPublicJwk) throws -> Bool {
+    public func verify(using verifier: TokenVerifying, withPublicKey key: JWK) throws -> Bool {
         return try verifier.verify(token: self, usingPublicKey: key)
     }
     
+    /// Temporary: TODO: remove support for ECPublicJwk data model for JWK.
```

Comment:
Expected? Do we have a bug tracking this?

Reply:
bug added.